src/org/labkey/targetedms/TargetedMSServiceImpl.java

```diff
@@ -116,6 +120,43 @@ public List<SampleFileInfo> getSampleFiles(Container container, User user, Integ
         return TargetedMSManager.get().getSampleFileInfos(container, user, sampleFileLimit);
     }
 
+    @Override
+    public List<SampleFileInfo> getSampleFiles(Container container, User user, Date startDate, Date endDate)
+    {
+        List<SampleFileInfo> sampleFileInfos = TargetedMSManager.get().getSampleFileInfos(container, user, null);
+        List<SampleFileInfo> result = new ArrayList<>();
+        if (!sampleFileInfos.isEmpty())
+        {
+            for (SampleFileInfo sampleFileInfo : sampleFileInfos)
+            {
+                DateFormat format = DateFormat.getDateInstance();
+                String acquiredTime = format.format(sampleFileInfo.getAcquiredTime());
```

Comment:
Why do we format this as a string and then immediately parse it back to a Date?

Reply:
I was doing this to get the date only part. Thanks for the pointer, replaced it with DateUtil.getDateOnly